src/providers/aws.rs

```diff
@@ -83,6 +83,26 @@ pub struct No;
 #[derive(Clone, Copy, Debug)]
 pub struct Yes;
 
+/// Available configurations of availability zone specifiers.
+///
+/// See [the aws docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#using-regions-availability-zones-launching) for more information.
+#[derive(Debug, Clone)]
+pub enum AvailabilityZoneSpec {
+    /// `Any` (the default) will place the instance anywhere there is capacity.
+    Any,
+    /// `Cluster` will place all the instances with the given id together, without specifying
```

Comment:
this might benefit from a little bit of elaboration--I don't totally get the "given ID"/usize thing